### PR TITLE
Update ota.md: Add note for Pixel 6/7 users OTA

### PR DESCRIPTION
**Pull request by n0k0m3**
Add note for Pixel 6/7 users OTA to reflect current status of https://github.com/topjohnwu/Magisk/issues/5959#issuecomment-1152868270

### CODE REVIEW

File: docs/ota.md [hunk 1]
```diff
@@ -14,6 +14,9 @@ Magisk does not modify most read-only partitions, which means applying official
 
 ### Devices with A/B Partitions
 
+**Notes:** For the moment `Install to Inactive Slot` method won't work for all Pixel 6 and 7 models (https://github.com/topjohnwu/Magisk/issues/5959). Either follow ["Non A/B" Devices](#non-ab-devices) guide below or use a 3rd party tool like [PixelFlasher](https://github.com/badabing2005/PixelFlasher)
```
Comment (SammCheese) [inline docs/ota.md:17]: Avoid linking 3rd-party-tools. Even if they are popular/safe atm, it can change at any point.

Comment (n0k0m3) [reply docs/ota.md:17]: Does saying "use other 3rd party pixel rooting/flashing tool" without explicitly mentioning PixelFlasher or name of other tool work?

Comment (SammCheese) [reply docs/ota.md:17]: Aye

**Comment by vvb2060** [comment]
We don't know the definitive compatibility list, and why some devices are available.  So let it be as it is.

**Pull request closed without merging**